inst/mlr3shiny/server/server_learner.R

```diff
@@ -709,7 +709,9 @@ makeLearner <- function(learnerobject, learnername, trigger, selectedlearner, le
          paramlist[[paste0(learnerobject$Learner_Name, ".", "type")]] <- "eps-regression"
       }
 
-      learnerobject$Learner$param_set$values <- paramlist # update hyperparameter values of current learner
+      #old: learnerobject$Learner$param_set$values <- paramlist # update hyperparameter values of current learner
+      learnerobject$Learner$param_set$values <- c(learnerobject$Learner$param_set$values, paramlist) # update hyperparameter values of current learner
```

Comment:
We would rather need some sort of set operation instead of making it a long vector that could potentially hold multiple values for the same hyperparameter